### PR TITLE
III-4475 - Add tabs additional information

### PR DESCRIPTION
**Pull request by simon-debruijn**
### Changed

- Move steps of AdditionalInformation under tabs
- Move DescriptionStep into separate component/file

### Fixed

- Fix description input lagging by moving DescriptionStep into separate component/file

---

Ticket: https://jira.uitdatabank.be/browse/III-4475

⚠️ In the next PR I will separate all steps of AdditionalInformation, like I did here for Description. My goal will also be to clean up tabsConfigurations, since right now it has a large dependency array.

Screenshot 2022-06-24 at 14 33 41

### CODE REVIEW

**Review by geoffreydhuyvetters — APPROVED** [review]

File: src/pages/steps/AdditionalInformationStep.tsx [hunk 1]
```diff
@@ -64,10 +57,33 @@ type Field =
...
   | 'priceInfo'
   | 'audience'
   | 'organizer';
 
+type TabConfig = {
+  eventKey: string;
+  title: string;
+  Component: ReactNode;
+  visible: boolean;
+  completed: boolean;
```
Comment (geoffreydhuyvetters) [inline src/pages/steps/AdditionalInformationStep.tsx:70]: isCompleted

File: src/pages/steps/AdditionalInformationStep.tsx [hunk 2]
```diff
@@ -64,10 +57,33 @@ type Field =
...
+  | 'contactInfo'
   | 'priceInfo'
   | 'audience'
   | 'organizer';
 
+type TabConfig = {
+  eventKey: string;
+  title: string;
+  Component: ReactNode;
+  visible: boolean;
```
Comment (geoffreydhuyvetters) [inline src/pages/steps/AdditionalInformationStep.tsx:69]: isVisible

File: src/pages/steps/AdditionalInformationStep.tsx [hunk 3]
```diff
@@ -64,10 +57,33 @@ type Field =
...
+
+const TabTitle = ({ title, completed, ...props }: TabTitleProps) => {
+  return (
+    <Inline spacing={3} {...getInlineProps(props)}>
+      {completed && <Icon name={Icons.CHECK_CIRCLE} color="#48874a" />}
+      <Text>{title}</Text>
+    </Inline>
+  );
+};
+
```
Comment (geoffreydhuyvetters) [inline src/pages/steps/AdditionalInformationStep.tsx:86]: Is there a specific benefit of extracting this to a separate component? Feels like we're just creating an extra layer of abstraction for the sake of it here.

**Review by geoffreydhuyvetters — APPROVED** [review]

File: src/pages/steps/AdditionalInformationStep.tsx (hunk 1 above)
Comment (brampauwelyn) [reply src/pages/steps/AdditionalInformationStep.tsx]: ✅

File: src/pages/steps/AdditionalInformationStep.tsx (hunk 3 above)
Comment (brampauwelyn) [reply src/pages/steps/AdditionalInformationStep.tsx]: I don't know the exact reason.
I'll have to check with @simon-debruijn

**Pull request merged**